plugins/copr.py

```diff
@@ -148,6 +148,8 @@ def run(self):
 
         repo_filename = "{}/_copr_{}-{}.repo" \
                         .format(self.base.conf.get_reposdir, copr_username, copr_projectname)
+        modules_repo_filename = "/etc/fm.modules.d/_copr_{}-{}.cfg" \
```

Comment:
Would something like this be preferable over the current solution where /etc/fm.modules.d is dynamically created if it does not exist yet?

[0001-added-etc-fm.modules.d.patch.txt](https://github.com/rpm-software-management/dnf-plugins-core/files/951629/0001-added-etc-fm.modules.d.patch.txt)

(Please, consider this to be just a proof of concept)